week_1/css_advanced_and_best_practices/portfolio-styles.css

```diff
@@ -69,20 +63,22 @@ form label {
   margin-bottom: 0.5rem;
   color: #333;
 }
-
 form input,
 form textarea {
   width: 100%;
+  height: 100%;
   padding: 0.5rem;
   margin-bottom: 1rem;
   border: 1px solid #ccc;
   border-radius: 5px;
```

Comment:
**Suggestion:** The height property in form input and textarea is set to 100%, which may cause the input fields to take up the entire height of the viewport. It's better to set a specific height or remove the height property.
```suggestion
form input,
form textarea {
  width: 100%;
  padding: 0.5rem;
  margin-bottom: 1rem;
  border: 1px solid #ccc;
  border-radius: 5px;
}
```